### ISSUE TITLE
Typo in Watershed.connect() example in README.md

### ISSUE THREAD

**Issue by RnMss** [joyent/node-watershed#6]
In the example for `Watershed.connect`

```javascript
var shed = new Watershed();
var wskey = ws.generateKey();
var options = {
    port: 8082,
    hostname: '127.0.0.1',
    headers: {
        'connection': 'upgrade',
        'upgrade': 'websocket',
        'Sec-WebSocket-Key': wskey,
        'Sec-WebSocket-Version': '13'
    }
};
var req = http.request(options);
req.end();
req.on('upgrade', function(res, socket, head) {
        var wsc = shed.connect(res, socket, head, wskey);
        wsc.send('Hi there!');
        wsc.on('end', function() {
                console.log('end!');
        });
        /* ... etc ... */
});
```

What's this variable `ws`, in line 2?

It is a typo which should actually be `shed.generateKey()`?

**Comment by melloc** [comment]
@RnMss Thanks for noticing this! That should be `shed` instead of `ws`. I'll go ahead and fix the example.

**Comment by melloc** [comment]
Fixed in d177805.
